### PR TITLE
CAF-2752 audit doc site

### PR DESCRIPTION
**Pull request by glucy**
Site can be viewed here:  https://pages.github.hpe.com/sepg-docs-qa/CAFAudit_CI_audit-service_CAF-2752_Audit_Doc_Site/

### CODE REVIEW

File: docs/_posts/2017-05-03-CAF_Audit_V3.0.0_Released.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+author: Greg Lucy
+datestamp: May 3rd 2017 - 16:21pm
+---
+
+Audit Service V3.0.0 Released!
+We are proud to announce the release of version 3.0.0 of the Audit Service.
+
+**Key Features:**
+
+- The Audit Service now uses Elasticsearch!
```
Comment (CJComac) [inline docs/_posts/2017-05-03-CAF_Audit_V3.0.0_Released.md]: Might be worth adding a bullet point that lists that our Audit service is now open sourced. It probably should have been added to the blog on release of version 2.0.0

File: docs/index.md [hunk 2]
```diff
@@ -9,24 +9,21 @@ button:
...
     feature_list:
         - image: assets/img/LandingIcons/support.png
           title: Support for Multiple Applications
-          subtitle: User and system audit events are defined per application. The Audit service offers an easy and user-friendly way to register these events using the Audit Management web service API.
+          subtitle: User and system audit events are defined per application.
         - image: assets/img/LandingIcons/multi-tenant.png
           title: Multi-Tenant Aware
-          subtitle: The service supports multiple tenants in a secure and scalable way. These tenants are registered using the Audit Management web service API. Multi-tenancy support also facilitates audit reporting on a per tenant basis.
+          subtitle: The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis.
```
Comment (CJComac) [inline docs/index.md]: Given that a user can construct multi-index queries with the ES Search API, https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html , I think this should read "The service supports multiple tenants in a secure and scalable way. Multi-tenancy support also facilitates audit reporting across multiple tenants."

File: docs/pages/en-us/Features.md [hunk 3]
```diff
@@ -13,29 +13,24 @@ banner:
...
+The Audit service maintains a documented audit trail of user and system activity in a centralized database. The benefits of using the Audit service include traceability and accountability, but the audit records can also be used for analytical, archiving and reporting purposes.
 
 ## Support for Multiple Applications
-User and system audit events are defined per application. The Audit service offers an easy and user-friendly way to register these events using the Audit Management web service API.
+User and system audit events are defined per application.
 
 ## Multi-Tenant Aware
-The service supports multiple tenants in a secure and scalable way. These tenants are registered using the Audit Management web service API. Multi-tenancy support also facilitates audit reporting on a per tenant basis. 
+The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis.
```
Comment (CJComac) [inline docs/pages/en-us/Features.md]: _Same comment as the landing page:_ Given that a user can construct multi-index queries with the ES Search API, https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html , I think this should read "The service supports multiple tenants in a secure and scalable way. Multi-tenancy support also facilitates audit reporting across multiple tenants."

File: docs/pages/en-us/Features.md [hunk 4]
```diff
@@ -13,29 +13,24 @@ banner:
...
 ## Auto-Generated Client Library
-The service includes an auto-generated client library for type safety. This client library also makes it easier to send user and system audit events to the messaging system.
-
-## High-Throughput Messaging
-The service uses Apache Kafka internally. Apache Kafka's high-throughput distributed messaging system is designed to be fast.
+The service includes an auto-generated client-side library for type safety.  This client library also makes it easier to send user and system audit events to Elasticsearch.
 
 ## Scalable
-The service is designed for scalability as the Kafka cluster can elastically and transparently expand without downtime. Data streams are partitioned over a cluster of machines to provide streams much larger than those available through a single machine.
+The service is extensively for scalabe as the Elasticsearch is designed with scalability in mind and can scale horizontally without downtime.
```
Comment (CJComac) [inline docs/pages/en-us/Features.md]: Grammar and spelling correction here. I would change it to something like: "The service is extensively scalable. Elasticsearch is designed with scalability in mind and can scale horizontally without downtime."

File: docs/pages/en-us/overview.md [hunk 5]
```diff
@@ -15,37 +15,22 @@ banner:
 
 The Audit service provides applications with a reliable, high-performance solution for recording events pertaining to user and system applications. 
 
-This documented audit trail of user and system activity has many uses. It provides traceability and individual accountability. Audit records can also be used for analytical purposes, including security violation detection and abnormal usage patterns. Audit trails have legal standing and can help protect the organization with a proven record of user and system activity. Audit records can also be exported for archival and reporting purposes.
+This documented audit trail of user and system activity has many uses.  It provides trace-ability and individual accountability.  Audit records can also be used for analytical purposes, including security violation detection and abnormal usage patterns.  Audit trails have legal standing and can help protect the organization with a proven record of user and system activity.  Audit records can also be exported for archival and reporting purposes.
```
Comment (CJComac) [inline docs/pages/en-us/overview.md]: I would unhyphenate "trace-ability"

File: docs/pages/en-us/overview.md [hunk 6]
```diff
@@ -15,37 +15,22 @@ banner:
...
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  An application audit event definition file can be used to generate an application-specific, client-side auditing library.  The Audit service is multi-tenant aware.  Applications send events using the generated client-side auditing library to Elasticsearch where they are indexed according to each tenant.
 
 ## User and System Actions
-Applications define user and system actions in an audit event definition file. This definition is registered using the Audit Management web service API to create the Vertica database schema for storing the audit event messages. The definition is also utilized to generate a Java SDK, which raises the defined audit events.
+Applications define user and system actions in an audit event definition file.
```
Comment (CJComac) [inline docs/pages/en-us/overview.md]: I would extend this a little to read: "Applications define user and system actions in an audit event definition file which is utilized to generate a client-side auditing library that raises the defined audit events."

File: docs/pages/en-us/overview.md [hunk 7]
```diff
@@ -15,37 +15,22 @@ banner:
...
 
 For more details on the audit events definition schema, see [Getting Started](Getting-Started).
```
Comment (CJComac) [inline docs/pages/en-us/overview.md]: Reword to "For more details on the audit events definition file, see ...". There is such a thing as the 'audit events definition schema' but its the mechanism that is used for IntelliSense and validation of audit events definition file xml elements. It sounds out of place here.

File: docs/pages/en-us/overview.md [hunk 8]
```diff
@@ -15,37 +15,22 @@ banner:
...
 
-## Java SDK
-The audit events definition file is used by a code generation plugin to auto-generate a Java library, which sends audit event messages to the Apache Kafka messaging service.
+## Audit Library
+The audit events definition file is used by a code generation plugin to auto-generate a client-side Java library, which provide type safety and sends audit event messages to Elasticsearch.
 
-For more details on the Auditing Java SDK, see [Client-API](Client-API).
+For more details on the Java Client-Side Audit Library, see [Client-API](Client-API).
```
Comment (CJComac) [inline docs/pages/en-us/overview.md]: This was originally wrong. As it currently stands, the Getting-Started guide is what covers the generating and usage of the client-side auditing library (aka the AuditLog class). The Client-side API page actually covers using the `caf-audit` library directly and the AuditEventBuilder that it offers. I would change this to point the user to the 'Getting-Started guide > Generating a Client-side Auditing Library' section.

File: docs/pages/en-us/overview.md [hunk 9]
```diff
@@ -15,37 +15,22 @@ banner:
...
-
-To see the web methods in the Audit Management web service API, see [Server-API](Server-API).
-
-For more details on the architecture of the Audit Management web service API, see [Architecture](Architecture).
-
-For instructions on deploying and using the Audit Management web service API, see [Getting Started](Getting-Started).
+## Elasticsearch
+Elasticsearch is an opensource, distributable, scalable, enterprise grade search engine.  Elasticsearch is accessible through an extensive RESTful API and can provide multi-tenant capable full text searches that support data discovery.  Official clients are available in Java, .NET (C#), Python, Groovy and many other languages.
 
+For more details on Elasticsearch deployment, see [Audit Service Deploy Project in GitHub](https://github.com/CAFAudit/audit-service-deploy).
```
Comment (CJComac) [inline docs/pages/en-us/overview.md]: I would include a link to Elasticsearch's homepage too for readers that need more details on it.

File: docs/index.md (hunk 2 above)
Comment (glucy) [reply docs/index.md]: Updated to: "The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis and across multiple tenants"

File: docs/_posts/2017-05-03-CAF_Audit_V3.0.0_Released.md (hunk 1 above)
Comment (glucy) [reply docs/_posts/2017-05-03-CAF_Audit_V3.0.0_Released.md]: Agreed. Added opensourcing point.

File: docs/pages/en-us/Features.md (hunk 3 above)
Comment (glucy) [reply docs/pages/en-us/Features.md]: Updated to the same as above: "The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis and across multiple tenants"

File: docs/pages/en-us/Features.md (hunk 4 above)
Comment (glucy) [reply docs/pages/en-us/Features.md]: No idea what I was trying to write.  Agreed and updated.

File: docs/pages/en-us/overview.md (hunk 5 above)
Comment (glucy) [reply docs/pages/en-us/overview.md]: Agreed, removed hyphen.

File: docs/pages/en-us/overview.md (hunk 6 above)
Comment (glucy) [reply docs/pages/en-us/overview.md]: OK.  Had stripped out the "generation" as it is mentioned again in the next point, but I see the existing documentation mentions it twice so I've added it back following your suggestion.

"Applications define user and system actions in an audit event definition file.  The definition is utilized to generate a client-side auditing library which raises the defined audit events."

File: docs/pages/en-us/overview.md (hunk 7 above)
Comment (glucy) [reply docs/pages/en-us/overview.md]: Reworded as suggested.

File: docs/pages/en-us/overview.md (hunk 8 above)
Comment (glucy) [reply docs/pages/en-us/overview.md]: Updated link to the new "Generating a Client-side Auditing Library" anchor.

File: docs/pages/en-us/overview.md (hunk 9 above)
Comment (glucy) [reply docs/pages/en-us/overview.md]: Added:  For more details on Elasticsearch, see [Elasticsearch](https://www.elastic.co/products/elasticsearch).

**Review by glucy — COMMENTED** [review]
Changes made following review comments

File: docs/index.md [hunk 10]
```diff
@@ -9,24 +9,21 @@ button:
 
 features:
     title: About CAF Auditing
-    subtitle: The Audit service maintains a documented audit trail of user and system activity in a centralized database. The benefits of using the Audit service include traceability and accountability, but the audit records can also be used for analytical, archiving and reporting purposes.
+    subtitle:  The Audit service maintains a documented audit trail of user and system activity in a single resource.  The benefits of using the Audit service include traceability and accountability, but the audit records can also be used for analytical, archiving and reporting purposes.
```
Comment (glucy) [inline docs/index.md:12]: Replaced: "a centralized database" with "a single resource"

**Pull request merged**